api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -265,6 +277,12 @@ message ExternalProcessor {
   google.protobuf.Duration deferred_close_timeout = 19;
 }
 
+// ExtProcHttpService is used for raw HTTP communication between the filter and the external processing service.
```

Comment:
```suggestion
// ExtProcHttpService is used for HTTP communication between the filter and the external processing service.
```

"raw" doesn't serve any purpose here and is ambiguous.

Reply:
done